spec/requests/api/docs/v3/applications_spec.rb

```diff
@@ -71,19 +71,25 @@
         end
       end
     end
-  end
 
-  it_behaves_like "an API update endpoint documentation",
-                  "/api/v3/npq-applications/{id}/change-funded-place",
-                  "NPQ Applications",
-                  "Change funded place value of an NPQ application",
-                  "The NPQ application after changing the funded place",
-                  "#/components/schemas/ApplicationResponse",
-                  "#/components/schemas/ApplicationChangeFundedPlaceRequest" do
-    let(:application) { create(:application, :eligible_for_funded_place, lead_provider:, schedule:, cohort:, course:) }
-    let(:resource) { application }
-    let(:type) { "npq-application-change-funded-place" }
-    let(:attributes) { { funded_place: true } }
-    let(:invalid_attributes) { { funded_place: nil } }
+    it_behaves_like "an API update endpoint documentation",
+                    "/api/v3/npq-applications/{id}/change-funded-place",
+                    "NPQ Applications",
+                    "Change funded place value of an NPQ application",
+                    "The NPQ application after changing the funded place",
+                    "#/components/schemas/ApplicationResponse",
+                    "#/components/schemas/ApplicationChangeFundedPlaceRequest" do
+      let(:application) { create(:application, :eligible_for_funded_place, lead_provider:, schedule:, cohort:, course:) }
+      let(:resource) { application }
+      let(:type) { "npq-application-change-funded-place" }
+      let(:attributes) { { funded_place: true } }
+      let(:invalid_attributes) { { funded_place: nil } }
+      let(:response_example) do
+        base_response_example.tap do |example|
+          example[:data][:attributes][:status] = "accepted"
+          example[:data][:attributes][:funded_place] = true
```

Comment:
could we also this on the accepted endpoint and set the `funded_place` to true :)?